src/main/java/com/endava/soa_5ed/task2/arrays/MaxNumberIdentifier.java

```diff
@@ -0,0 +1,17 @@
+package com.endava.soa_5ed.task2.arrays;
+
+public class MaxNumberIdentifier {
```

Comment:
you may add here a property : private int[] array; 
a constructor that will initialize you property; 
and you may remove the copy-pasted method parameter from 5 an 9 line, an use newly added property.

Reply:
done, thank you